.github/workflows/build.yml

```diff
@@ -33,6 +33,8 @@ jobs:
           # it relies on the existence of a go.sum file.
           cache: false
           go-version: "1.20"
+      - id: setup-rust
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
Why not use [the more official GH Action](https://github.com/marketplace/actions/rust-toolchain)?

Reply:
Looking at the repo it's very dead. The last release was in 2020.